pkg/config/config.go

```diff
@@ -1539,7 +1541,16 @@ func setupFipsEndpoints(config Config) error {
 	os.Unsetenv("HTTPS_PROXY")
 
 	// HTTP for now, will soon be updated to HTTPS
-	protocol := "http://"
+	var protocol string
+	if config.GetBool("fips.https") {
```

Comment:
I think `logs_config.logs_no_ssl` and `logs_config.use_http` below may need changing to support this logic too

Reply:
I have to speak to Maxime about this. I don't understand why he had structured this code this way.